backend/src/main/java/eu/bbmri_eric/negotiator/notification/email/EmailTemplateController.java

```diff
@@ -0,0 +1,72 @@
+package eu.bbmri_eric.negotiator.notification.email;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.security.SecurityRequirement;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.List;
+import java.util.Map;
+import lombok.extern.apachecommons.CommonsLog;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+@RestController
+@RequestMapping("/v3")
+@CommonsLog
+@Tag(name = "Email Templates", description = "Retrieve and update email templates")
+@SecurityRequirement(name = "security_auth")
+public class EmailTemplateController {
+
+  @Autowired EmailTemplateService emailTemplateService;
+
+  @GetMapping("/email-templates")
+  @Operation(
+      summary = "Get all notification email-templates",
+      description = "Get all notification email-templates")
+  @ResponseStatus(HttpStatus.OK)
+  public List<String> getAllNotificationTemplates() {
+    return emailTemplateService.getAllNotificationTemplates();
+  }
+
+  @GetMapping(
+      value = "/email-templates/{templateName}",
+      produces = MediaType.APPLICATION_XHTML_XML_VALUE)
+  @Operation(
+      summary = "Get a notification email-template",
+      description = "Get a notification email-template by name")
+  @ResponseStatus(HttpStatus.OK)
+  public String getNotificationTemplate(@PathVariable String templateName) {
+    return emailTemplateService.getNotificationTemplate(templateName);
+  }
+
+  @PostMapping(
+      value = "/email-templates/{templateName}",
+      produces = MediaType.APPLICATION_XHTML_XML_VALUE)
+  @Operation(
+      summary = "Update a notification email-template",
+      description = "Update the content of a notification email-template by name")
+  @ResponseStatus(HttpStatus.OK)
+  public String updateNotificationTemplate(
+      @PathVariable String templateName, @RequestBody String template) {
+    return emailTemplateService.updateNotificationTemplate(templateName, template);
+  }
+
+  @PatchMapping(
```

Comment:
not sure this is the best approach, patch is used to update a specific field of a Resource (this does not seem to be the case). Either we go back to /reset which breaks our commitment to keep the API RESTful, or we do /email-templates/name/operations which accepts a DTO that has only one attribute enum "value" that currently only supports RESET.